docs/build/messages/read-receipt.mdx

```diff
@@ -246,8 +251,8 @@ Read receipts for Dart haven't been implemented yet
 <TabItem value="rn" label="React Native"  attributes={{className: "rn_tab"}}>
 
 ```jsx
-if (message.contentTypeId !== "xmtp.org/readReceipt:1.0") {
-  throw Error("Unexpected message content " + bobMessages[0].content);
+if (message.contentTypeId === "xmtp.org/readReceipt:1.0") {
+  return message.content().timestamp; //Date received
```

Comment:
Read receipt content does not have anything on it. You need to get the timestamp from the message itself.

```suggestion
  return message.sent; //Date received
```